Formula/kdoctor.rb

```diff
@@ -0,0 +1,26 @@
+class Kdoctor < Formula
+  desc "Environment diagnostics for Kotlin Multiplatform Mobile app development"
+  homepage "https://github.com/kotlin/kdoctor"
+  url "https://github.com/Kotlin/kdoctor/archive/refs/tags/v0.0.2.tar.gz"
+  sha256 "e95af8f76fb3a6240758af76b5528e57b41793065354c81bff2fbebe435f22df"
+  license "Apache-2.0"
+  head "https://github.com/Kotlin/kdoctor.git", branch: "master"
+
+  depends_on "gradle" => :build
+  depends_on "openjdk" => :build
+  depends_on xcode: ["12.5", :build]
+  depends_on :macos
+
+  def install
+    ENV["JAVA_HOME"] = Formula["openjdk"].opt_prefix
+    mac_suffix = Hardware::CPU.intel? ? "X64" : Hardware::CPU.arch.to_s.capitalize
+    build_task = "linkReleaseExecutableMacos#{mac_suffix}"
+    system "gradle", "clean", build_task
+    bin.install "kdoctor/build/bin/macos#{mac_suffix}/releaseExecutable/kdoctor.kexe" => "kdoctor"
+  end
+
+  test do
+    r = /(.*)(System)(.*)(Java)(.*)(Android Studio)(.*)(Xcode)(.*)(Cocoapods)(.*)/m
+    assert_match r, shell_output("#{bin}/kdoctor")
```

Comment:
```suggestion
    output = shell_output(bin/"kdoctor")
    assert_match "System", output
    assert_match "Java", output
    assert_match "Android Studio", output
    assert_match "Xcode", output
    assert_match "Cocoapods", output
```
Let's try this, so we don't have to carry around a `Regexp` object.

Reply:
ok, lets do this way